src/2982R1_std_quantity_as_a_numeric_type.md

```diff
@@ -1907,13 +1907,14 @@ inline constexpr struct hubble_constant :
 Another important use case for dimensionless quantities is to provide strong types for counts
 of things. For example:
 
-- ISO-80000-3 provides a `rotation` quantity defined as the number of revolutions,
-- IEC-80000-6 provides a `number_of_turns_in_a_winding` quantity,
-- IEC-80000-13 provides a `Hamming_distance` quantity defined as the number of digit positions
+- [@ISO80000] (part 3) provides a `rotation` quantity defined as the number of revolutions,
+- [@ISO80000] (part 6) provides a `number_of_turns_in_a_winding` quantity,
+- [@ISO80000] (part 13) provides a `Hamming_distance` quantity defined as the number of digit positions
   in which the corresponding digits of two words of the same length are different.
 
-Thanks to assigning strong names to such quantities, later on they can be explicitly used as
-arguments in the quantity equations of other quantities deriving from them.
+Thanks to assigning strong names to such quantities,
+they can be used in the quantity equation of other quantities.
+For example, `rotational_frequency` is defined by `rotation / duration`.
```

Comment:
"Deriving from them" is simply too confusing.
It's not the C++ term related to base and derived classes.
It's more like "they're used in the definition of new quantities".